### ISSUE TITLE
Handle creating kafka topics.

### ISSUE THREAD

**Issue by tribeiro** [tribeiro/rs_salobj#42]
While working in #39 I realized that the [kafka library](https://github.com/kafka-rust/kafka-rust) I have been using has a major hole, apparently, it does not handle topic creation. From what I have understood so far it seems like they where relying on automatic topic creation, which was default in previous versions of kafka but was changed recently. Our systems will run with this configuration so the library must be able to handle topic creation, which probably means I will need to use a different kafka library.

It seems like the only viable alternative is [rust-rdkafka](https://github.com/fede1024/rust-rdkafka), which unfortunately, is an interface to the C++ library.

**Comment by tribeiro** [comment]
After writing this ticket I realized I was actually wrong. The Kafka-rust library does support topic creating you just need to make sure you pool for the information afterwards. I found the solution is this issue https://github.com/kafka-rust/kafka-rust/issues/135. Basically something like this:

```
    let mut client = KafkaClient::new(brokers);

    let mut attempt = 0;
    loop {
        attempt += 1;
        let _ = try!(client.load_metadata(&[topic]));
        if client.topics().partitions(topic).map(|p| p.len()).unwrap_or(0) > 0 { // <-- HERE
            break;
        } else if attempt > 2 { // try up to 3 times
            // return some error
            return Err(KafkaError::Kafka(KafkaCode::UnknownTopicOrPartition));
        }
        thread::sleep(Duration::from_secs(1));
    }
```


I will keep this ticket open for now but I'll try to implement this as part of #39 . If that works ok, I'll close it.

**Comment by tribeiro** [comment]
Fixed  in #39
